web/src/components/molecules/MyIntegration/WebhookList/index.tsx

```diff
@@ -0,0 +1,76 @@
+import styled from "@emotion/styled";
+
+import Button from "@reearth-cms/components/atoms/Button";
+import Icon from "@reearth-cms/components/atoms/Icon";
+import { useT } from "@reearth-cms/i18n";
+
+import WebhookCard from "./WebhookCard";
+
+export type Props = {
+  webhooks?: any[];
+};
+
+const WebhookList: React.FC<Props> = ({ webhooks }) => {
+  const t = useT();
+  return (
+    <>
+      <ActionWrapper>
+        <Button onClick={() => {}} type="primary" icon={<Icon icon="plus" />}>
+          {t("New Webhook")}
+        </Button>
+      </ActionWrapper>
+      {webhooks && webhooks.length > 0 ? (
+        <ListWrapper>
+          <WebhookCard title="Item_Hook" url="https://reearth.io/itemchanged" />
```

Comment:
```suggestion
          <WebhookCard title=t("Item_Hook") url="https://reearth.io/itemchanged" />
```
If this title final please wrap it in `t()`.

Reply:
@nourbalaha , it will be dynamic data, I added these for demonstration.

Reply:
I see, so I will resolve this conversation as dynamic data does not need to be wrapped in `t()`.